dependencies-jammy/Dockerfile

```diff
@@ -76,6 +76,31 @@ RUN cd /usr/src && \
     cmake -GNinja .. -DCMAKE_INSTALL_PREFIX=/usr/ && \
     ninja install && cd ../ && rm -rf build 
 
+# VTK
+RUN cd /usr/src && \
```

Comment:
For the jammy release, `VTK 9.1` packages exists, see [here](https://packages.ubuntu.com/jammy/libvtk9-dev).

Do you require a custom installation of VTK? If not, I would suggest to opt for the package on jammy. This will shorten the build time of the docker image.

Reply:
Hi @marcfehling, thanks for pointing this out!

Unfortunately in Ubuntu 22.04 there is a clash between the dependencies of Trilinos and VTK, please see the discussion [here](https://github.com/dealii/dealii/pull/15266#issuecomment-1563047320).

Additionally, the custom build commands that I've provided result in a very lightweight installation of VTK, by getting rid of all the graphical modules that will likely never be used in deal.II.

Tested on my local machine and using 2 cores, the additional time for building the Docker associated with VTK is around 20 minutes.

Reply:
Thanks for pointing it out. That's annoying! Looks like the difference between `libtbb2` and `libtbb` is really just the version `tbb 2020.3` vs `tbb 2021.5` (probably for backwards compatibility?).

You found a working solution that works, and I guess 20 minutes extra for building the image is okay. Not ideal, but okay.